frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,74 @@
+import { useRef } from "react";
+import './Signup.css';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const Login = ({setCurrUser, setShow}) =>{
+  const formRef=useRef();
+  const login=async (userInfo, setCurrUser)=>{
+    const url="http://localhost:3000/login";
+    try{
+        const response=await fetch(url, {
```

Comment:
We can use `Axios` for HTTP requests later.

Reply:
Yes Done sir.